src/guiFormSpecMenu.cpp

```diff
@@ -953,8 +962,10 @@ void GUIFormSpecMenu::parsePwdField(parserData* data,std::string element)
 		evt.KeyInput.PressedDown = true;
 		e->OnEvent(evt);
 
-		if (parts.size() >= 5 && !is_yes(parts[4])) {
-			spec.close_on_enter = false;
+		if (parts.size() >= 5) {
+			warningstream << "pwdfield: use field_close_on_enter[name, enabled]" <<
```

Comment:
This can be kept in to help modders do the transition, but should be removed one month or so after the PR got merged.